spec/models/rules/active_in_cohort_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe Rules::ActiveInCohort, type: :model do
+  before { skip "Requires warehouse classes" }
```

Comment:
Since these check against the local to CAS `active_cohort_ids` do we really need to skip all of them?

Reply:
I see, it's comparing against the ids, which probably wants a DB backing.... maybe `let(:cohort_a)..

Reply:
Replaced reified warehouse objects with simple ids to avoid dependency.